<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QDialog" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_espaceUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bienvenue dans votre espace utilisateur</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_user_deconnexion">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Deconnexion</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Selectionnez un profil</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Profil 1</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_validerAjoutSuppression">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_tables">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Affichage table</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>259</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>259</width>
     <height>264</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2_affichageDB">
      <property name="text">
       <string>Afficher la liste des bases de donn√©es</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView_dataBases"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_ajoutRecherche"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_AjoutRecherche">
        <item>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suprimer</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>331</width>
     <height>75</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
